docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,115 @@
+        - The fields `MattermostDownloadURL` and `LoadTestDownloadURL` point to the latest `mattermost-server`, and `load-test package`, to be used in the load-test.
+        - That's the default option, when there's unmerged changes to `mattermost-server`, or `mattermost-load-test-ng`.
+            - Run `make build-linux` in the `mattermost-server` directory, change the `MattermostDownloadURL` value to the path containing the Mattermost executable. For example `file:///somepath/mattermost-server/bin/linux_amd64/mattermost`.
+            - Run `make package` in the `mattermost-load-test-ng` directory, change `LoadTestDownloadURL` value to the path containing the gzip of the load-test package. For example `file:///somepath/mattermost-load-test-ng/dist/v1.5.0-8-gd4f18cf/mattermost-load-test-ng-v1.5.0-8-gd4f18cf-linux-amd64.tar.gz`.
+    - Edit `SSHPublicKey` in `deployer.json` after setting up ssh.
+    - `go run ./cmd/ltctl deployment create`
+        - Limit operations of `deployment` to a single shell window.
+        - If the deployment gets stuck, [check if AWS credentials are expired, and add new ones.](https://community.mattermost.com/core/pl/weau31yyp38btddryjuxbsnh1r).
+        - [Terraform actions are idempotent](https://community.mattermost.com/core/pl/jtebkneah3futd1y7pj8y9nrqy), so you rarely have to [destroy](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#destroy-the-current-deployment) the deployment, if things go wrong while creating resources.
+    - Once the deployment is successful, stdout will contain information on server addresses for app server, agent, coordinator, and Grafana deployments.
+        - Open the Mattermost URL in browser to confirm the app is working.
+        - At this point, you can check the server logs by [ssh-ing](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#ssh-access-to-the-terraformed-hosts) into the app instance. Once there, open `/opt/mattermost/logs/mattermost.log`.
+    - Gearing up to start the load-test:
+        - Use `agents'` URL and Prometheus URL in the `coordinator.json` file generated [here](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#copy-default-config).
+        - Change `ConnectionConfiguration` in the `config.json` generated [here](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#copy-default-config-1).
+        - Configure `InstanceConfiguration` in the same `config.json` file (which, as mentioned earlier, populates the Mattermost server's database with required data for the load test). Note that a heavier config with `NumPosts` would take a very long time get populated. Please refer to the [NB](/#nb) section below to manually populate the database from a backup, in order to bypass 'data-generation.
```

Comment:
I did not do this in my tests :eyes: Is this needed, @streamer45?

Reply:
If you use the mostly automated approach through Terraform there's no need to set URLs as they are generated by the deployer. But if you want to test something locally (or on some pre-existing installation) you'll have to change those.

Reply:
Do you think I should change anything here, or remove the instructions?

Reply:
I'd say that we can just remove these instructions altogether, since this section is to run an automated Terraform deployment, right, @plant99? I have a ticket to document how to load-test an existing deployment https://mattermost.atlassian.net/browse/MM-45947, so we can cover that there. Thoughts?

Reply:
Agreed, fixed this.